### PR TITLE
Add python examples for API v3 Documentation

### PR DESCRIPTION
**Pull request by dibyaaaaax**

### CODE REVIEW

**Comment by dibyaaaaax** [comment]
Issue  #5935

File: docs/api/v3.rst [hunk 1]
```diff
@@ -598,13 +748,30 @@ Subproject create
...
+
+            import requests
+            import json
+            URL = 'https://readthedocs.org/api/v3/projects/pip/subprojects/'
+            TOKEN = '<token>'
+            HEADERS = {'Authorization' : 'token {}'.format(TOKEN)}
+            data = json.load(open('body.json', 'rb'))
+            response = requests.post(
+                URL,
+                json = data,
```
Comment (stsewd) [inline docs/api/v3.rst]:
```suggestion
                json=data,
```

File: docs/api/v3.rst [hunk 2]
```diff
@@ -936,11 +1231,22 @@ Environment Variable delete
 
     **Example request**:
 
-    .. sourcecode:: bash
+   .. tabs::
```
Comment (stsewd) [inline docs/api/v3.rst]:
```suggestion
    .. tabs::
```

Comment (dibyaaaaax) [reply docs/api/v3.rst]: sorry about that, must have missed the last one while checking. I fixed it.

**Pull request merged**